src/context.rs

```diff
@@ -128,6 +128,17 @@ impl<'a> Context<'a> {
 
         let right = arguments.is_present("right");
 
+        let mut width = term_size::dimensions()
+            .map(|(width, _)| width)
+            .unwrap_or_default();
+        if width == 0 {
+            width = arguments
+                .value_of("terminal_width")
+                .unwrap_or("0")
+                .parse()
+                .unwrap_or(0);
+        }
+
```

Comment:
I think the command line argument should take precedence here, and I'd prefer a saner default and something more functional like this:
```suggestion
        let width = arguments
            .value_of("terminal_width")
            .and_then(|w| w.parse().ok())
            .or_else(|| terminal_size().map(|(w, _)| w.0 as usize))
            .unwrap_or(80);
```